upp/classes/components.py

```diff
@@ -91,6 +92,7 @@ def from_config(cls, pp_cfg):
             region_cuts = Cuts.empty() if pp_cfg.is_test else Cuts.from_list(c["region"]["cuts"])
             region = Region(c["region"]["name"], region_cuts + pp_cfg.global_cuts)
             pattern = c["sample"]["pattern"]
+            equal_jets = c["sample"]["equal_jets"]
```

Comment:
Here I think I should instead have something like:

```
equal_jets = c["sample"].get("equal_jets", True)
```
To get the key from the dictionary, but if the key doesn't exist give a default `True` value?